tracker/view/add.py

```diff
@@ -199,4 +200,21 @@ def add_group():
 
     db.session.commit()
     flash('Added {}'.format(group.name))
+    missing_lib32_variant(pkgnames, group)
+
     return redirect('/{}'.format(group.name))
+
+
+def missing_lib32_variant(pkgnames, group):
+    for pkgname in pkgnames:
+        if 'lib32' in pkgname:
+            continue
+
+        lib32pkg = f'lib32-{pkgname}'
+        if not Package.query.filter(Package.name == lib32pkg).first():
+            continue
+
+        if CVEGroupPackage.query.filter(CVEGroupPackage.pkgname == lib32pkg, CVEGroupPackage.group == group).first():
```

Comment:
can we make this algorithm in a way that we need to issue the queries once for all matches and not in a loop? optimally it looks like a single query can retreive all information we need to iterate over

Reply:
I've just realized that we are creating the group here, so we don't really have to query the db! We can just query group.packages and check if it contains a lib32 package, we only need to verify if a variant exists in the repo however.